resources/css/launcher.scss

```diff
@@ -209,7 +209,13 @@ body {
 }
 
 .release-notes {
-    overflow-y: overlay;
+    overflow-y: scroll;
+
+    max-height: 70dvh;
+    @media screen and (max-height: 650px) {
+        max-height: 55dvh;
+    }
```

Comment:
Makes me wonder, if the max-height is something which is only helpful for the release notes dialog or for all generic dialogs.